src/main.rs

```diff
@@ -22,7 +25,7 @@ mod callbacks;
 mod lua;
 
 /// Callback to route wlc logs into env_logger
-extern "C" fn log_handler(level: LogType, message_ptr: *const libc::c_char) {
+extern "C" fn log_handler(level: LogType, message_ptr: *const i8) {
```

Comment:
Why'd you switch to a raw pointer to an i8? A char in C could be an unsigned or signed char, the standard doesn't specify ([source](http://stackoverflow.com/questions/2054939/is-char-signed-or-unsigned-by-default)). I see no reason why we shouldn't use `libc::c_char` here?